### DISCUSSION TITLE
Discussion: Focus on javascript (drop Java, .Net, PHP)

### DISCUSSION THREAD

**Discussion by coclav** [maxGraph/maxGraph#19] [Development]
I suggest that this repo focus on the javascript version of the package and we drop java / .net / php (actually php was dropped a while ago) in another repo (same github organisation if need be) so that a community can form freely around these tech.

If nobody objects by the end of April 2021 i'll pull the trigger

**Comment by tbouffard** [comment]
👍🏿 for the JavaScript mxGraph focus only 

As I said in https://github.com/jsGraph/mxgraph/issues/8#issuecomment-810992407
> IMHO, they should be removed from this repository and the code should be moved to dedicated repositories if someone is ready to maintain it. The java version hasn't been updated for a while. I personally only use it indirectly with tools like https://github.com/process-analytics/mxgraph-svg2shape or https://github.com/jgraph/svg2xml
